product/tests/file_for_tests_migrations.py

```diff
@@ -0,0 +1,4 @@
+TEST_DATA = [
```

Comment:
I don't think it's a good idea to have data defined under `tests` and then use it elsewhere (e.g. code or migrations).

As i mentioned before, I think it would be best to have it defined inside the migrations file and then imported on the test file.

As this has been going back and forth now for a few times with suggestions from @Omeramsc and myself, I think it will be best to wait for more feedback before changing things again (mainly, to avoid unnecessary work on your side).

Reply:
this is exactly what I asked to be done on the other patch...

Reply:
Fixed!